test/e2e_node/container_lifecycle_test.go

```diff
@@ -5905,6 +5905,133 @@ var _ = SIGDescribe(feature.SidecarContainers, "Containers Lifecycle", func() {
 				})
 			})
 		})
+
+	})
+	ginkgo.When("A sidecar container with startup probe fails initially", func() {
+		ginkgo.It("should continue probing and allow main container to start after sidecar recovers ", func(ctx context.Context) {
```

Comment:
can we replace sidecar (container) with restartable init (container) and main container with regular container for consistency?

s/sidecar/restartable init/g
s/main container/regular container/g